test/functional/corporate_information_pages_controller_test.rb

```diff
@@ -111,8 +113,7 @@ class CorporateInformationPagesControllerTest < ActionController::TestCase
     organisation = create(:organisation)
     organisation2 = create(:organisation, slug: "another_organisation")
     cip = create(:corporate_information_page, :unpublished, organisation:)
-    alternative_url = Whitehall.url_maker.root_url
-    cip.unpublishing.update!(redirect: true, slug: cip.slug, alternative_url:)
+    cip.unpublishing.update!(redirect: true, slug: cip.slug, alternative_url: "https://www.gov.uk")
```

Comment:
Is there a helper method available that returns this string, to avoid duplicating it here?